apps/dotcms-ui/src/app/view/components/dot-container-selector/dot-container-selector.module.ts

```diff
@@ -18,7 +17,6 @@ import { UiDotIconButtonModule } from '@components/_common/dot-icon-button/dot-i
         SearchableDropDownModule,
         DotPipesModule,
         UiDotIconButtonModule
-    ],
-    providers: [PaginatorService]
+    ]
```

Comment:
why moving the `PaginatorService` from the module to the component fix the issue?

Reply:
Because the theme selector and the container selector use the same instance of the PaginatorService and when you use one selector or the other leave the instance dirty.

We might have this problem is other places as well.